editor/src/components/canvas/controls/select-mode/move-utils.ts

```diff
@@ -132,7 +132,7 @@ export function dragComponent(
               roundedDragDelta,
             ),
             scale,
-          )
+          ).delta
```

Comment:
Is this a bug on production right now? If not, it looks like this is going to affect the existing behaviour

Reply:
I changed the function's signature

Reply:
Ah, I missed that :+1: